sale_order_type/models/sale.py

```diff
@@ -12,26 +12,29 @@ class SaleOrder(models.Model):
         string="Type",
         compute="_compute_sale_type_id",
         store=True,
-        readonly=True,
-        states={"draft": [("readonly", False)], "sent": [("readonly", False)]},
+        readonly=False,
+        states={
+            "sale": [("readonly", True)],
+            "done": [("readonly", True)],
+            "cancel": [("readonly", True)],
+        },
```

Comment:
Same here, why? :thinking:

Reply:
You can check https://github.com/OCA/sale-workflow/pull/1193/files#r489781694